<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_26">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>수학문제추출</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton1">
              <property name="text">
               <string>초기화</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton2">
              <property name="text">
               <string>입력</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_db">
              <property name="text">
               <string>DBTEST</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton4">
              <property name="text">
               <string>저장</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <widget class="QPlainTextEdit" name="plainTextEdit"/>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="4">
                   <widget class="QPushButton" name="pushButton_23">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QPushButton" name="pushButton_24">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QPushButton" name="pushButton_25">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="3">
                   <widget class="QPushButton" name="pushButton_26">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QPushButton" name="pushButton_27">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QPushButton" name="pushButton_28">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QPushButton" name="pushButton_29">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="4">
                   <widget class="QPushButton" name="pushButton_30">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QPushButton" name="pushButton_31">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="pushButton_32">
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <property name="spacing">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>&gt;&gt;&gt;&gt;&gt;&gt;&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>&gt;&gt;&gt;&gt;&gt;&gt;&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>&gt;&gt;&gt;&gt;&gt;&gt;&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_14">
                <item>
                 <widget class="QPlainTextEdit" name="plainTextEdit2"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_19">
                  <item>
                   <widget class="QRadioButton" name="radioButton1">
                    <property name="text">
                     <string>초등학교</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton2">
                    <property name="text">
                     <string>중학교</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton3">
                    <property name="text">
                     <string>고등학교</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>[ 학 년 ]</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox1"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox2"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_20">
                  <item>
                   <widget class="QLabel" name="label_17">
                    <property name="text">
                     <string> [ 정 답 ] </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit1"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_18">
                    <property name="text">
                     <string> [ 난 이 도 ] </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox3">
                    <item>
                     <property name="text">
                      <string>최상</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>상</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>중상</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>중</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>중하</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>하</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_15">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_21">
                    <item>
                     <widget class="QCheckBox" name="checkBox_11">
                      <property name="text">
                       <string>보기1)</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit2"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_22">
                    <item>
                     <widget class="QCheckBox" name="checkBox_12">
                      <property name="text">
                       <string>보기2)</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit3"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_23">
                    <item>
                     <widget class="QCheckBox" name="checkBox_13">
                      <property name="text">
                       <string>보기3)</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit4"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_24">
                    <item>
                     <widget class="QCheckBox" name="checkBox_14">
                      <property name="text">
                       <string>보기4)</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit5"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_25">
                    <item>
                     <widget class="QCheckBox" name="checkBox_15">
                      <property name="text">
                       <string>보기5)</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit6"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>테이블관리</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton5">
              <property name="text">
               <string>검색</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget">
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>단원명</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>정답</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>보기1</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>보기2</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>보기3</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>보기4</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>보기5</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>학교</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>학년</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>난이도</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_capture"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>내보내기</string>
    </property>
    <addaction name="action_pdf"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="text">
    <string>열기</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
  <action name="action_capture">
   <property name="text">
    <string>캡쳐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="action_pdf">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
